src/Microsoft.AspNetCore.Mvc.TagHelpers/Internal/FileVersionProvider.cs

```diff
@@ -111,7 +111,14 @@ public string AddFileVersionToPath(string path)
 
         private static string GetHashForFile(IFileInfo fileInfo)
         {
-            using (var sha256 = SHA256.Create())
+            HashAlgorithm sha256;
+#if NETSTANDARD1_6
+            sha256 = SHA256.Create();
+#else
+            sha256 = new SHA256Cng();
```

Comment:
This one came from the .NET crypto guys this time around. Either works to be honest